include/umf/memtarget.h

```diff
@@ -31,6 +32,13 @@ typedef enum umf_memtarget_type_t {
 umf_result_t umfMemtargetGetType(umf_const_memtarget_handle_t hMemtarget,
                                  umf_memtarget_type_t *type);
 
+/// \brief Get capacity of the memory target.
```

Comment:
capacity? you mean the size of memory? in bytes?

Reply:
Done